.github/workflows/ci.yaml

```diff
@@ -1,6 +1,9 @@
 name: Ansible Docsite CI
 
 on:
+  schedule:
+    # Daily 
+    - cron: "0 7 * * *"
```

Comment:
This seems fine but I got a useful comment in a similar PR: https://github.com/ansible/ansible-documentation/pull/1663

Consider a random minute that isn't divisible by 5 as an offset to run when the org activity is at its lowest, distributing the CI resource usage throughout the day.

Reply:
23 is prime number (not going to be hit on any of the */x cycles) in the middle of the hour. That should do for "random"